src/gcp/apphosting.ts

```diff
@@ -460,3 +516,53 @@ export async function ensureApiEnabled(options: any): Promise<void> {
   const projectId = needProjectId(options);
   return await ensure(projectId, API_HOST, "frameworks", true);
 }
+
+/**
+ * Generates the next build ID to fit with the naming scheme of the backend API.
+ * @param counter Overrides the counter to use, avoiding an API call.
+ * @return
+ */
+export async function getNextRolloutId(
+  projectId: string,
+  location: string,
+  backendId: string,
+  counter?: number,
+): Promise<string> {
+  const date = new Date();
+  const year = date.getUTCFullYear();
+  // Note: month is 0 based in JS
+  const month = String(date.getUTCMonth()).padStart(2, "0");
```

Comment:
So should you be adding 1?

Reply:
😱 I modified the code and dropped the +1. Good catch and thank god for comments.